create.php

```diff
@@ -87,6 +87,23 @@
         if ($db->query($schema))
             echo "Created table 'clients'.\n";
 
+        $schema = 'CREATE TABLE sysinfo (
+                    system_id STRING PRIMARY KEY,
```

Comment:
This was intentional, yeah.  I felt that, at least for the initial implementation, throwing away info on how often people launch the game is a good tradeoff for the complexity of aggregating and expiring old data.